programs/dynamic-bonding-curve/src/instructions/initialize_pool/ix_initialize_virtual_pool_with_spl_token.rs

```diff
@@ -136,6 +136,10 @@ pub fn handle_initialize_virtual_pool_with_spl_token<'c: 'info, 'info>(
     params: InitializePoolParameters,
 ) -> Result<()> {
     let config = ctx.accounts.config.load()?;
+
+    // validate min base fee
+    config.pool_fees.base_fee.validate_base_fee()?;
```

Comment:
We need to block partner to create a new config with min_base_fee < min_base_fee as well

Reply:
i already changed value of`MIN_FEE_NUMERATOR` & `MIN_FEE_BPS` to 0.25% (previous 0.1%). And in create config step already has validation for these min base fee.